bin/Download_scripts/topaz_get_mercator_phy_CMT.sh

```diff
@@ -0,0 +1,126 @@
+      echo "Already exist, skip downloading GLOBAL data for $dateT1 12:00:00"
+      exit1_st=0
+      exit2_st=0
+   else
+      echo "Try to download from CMEMS catalogue"
+      count_f=0
+      # Variables are found in different datasets
+      for foo in {1..4} 
+      do
+         if [ ${foo} -eq 1 ] ; then
+            cmt_variables=' -v zos'
+            cmt_dataset='cmems_mod_glo_phy_anfc_0.083deg_P1D-m'
+         elif [ ${foo} -eq 2 ] ; then
+            cmt_variables=' -v thetao'
+            cmt_dataset='cmems_mod_glo_phy-thetao_anfc_0.083deg_P1D-m'
+         elif [ ${foo} -eq 3 ] ; then
+            cmt_variables=' -v so'
+            cmt_dataset='cmems_mod_glo_phy-so_anfc_0.083deg_P1D-m'
+         else
+            cmt_variables=' -v uo -v vo'
+            cmt_dataset='cmems_mod_glo_phy-cur_anfc_0.083deg_P1D-m'
+         fi
+         #
+         echo " "
+         for attempts in 1 2 3  # Try five times (because of frequent connections timeouts)
+         do
+           date
+           #echo "copernicusmarine subset --force-download --overwrite --no-metadata-cache --dataset-id ${cmt_dataset}\
+           #${cmt_variables}  -t "${dateT1}T00:00:00" -T "${dateT1}T00:00:00" ${cmt_region} ${cmt_depth}\
+           #    --username $username --password $password  -o ${DATADIR} -f global_analysis_forecast_phy${foo}_${date1}.nc"
+           copernicusmarine subset --force-download --overwrite --no-metadata-cache --dataset-id ${cmt_dataset}\
+              ${cmt_variables}  -t "${dateT1}T00:00:00" -T "${dateT1}T00:00:00" ${cmt_region} ${cmt_depth}\
+               --username $username --password $password  -o ${DATADIR} -f global_analysis_forecast_phy${foo}_${date1}.nc  
+           #---
+           if [ -s ${DATADIR}/global_analysis_forecast_phy${foo}_${date1}.nc ] ; then
+               echo "SUCCESS downloading GLOBAL data for global_analysis_forecast_phy${foo}_${date1}.nc"
+               count_f=$(( $count_f + 1 ))
+               echo "count=$count_f"
+               break
+            else
+              echo "FAILURE downloading GLOBAL data for global_analysis_forecast_phy${foo}_${date1}.nc"
+              if [[ "${attempts}" < "3" ]] ; then
+                 echo "Attempt ${attempts}/5 failed. Waiting 60 s until next attempt."
+                 sleep 10
+              fi
+            fi
+         done
+      done
+      if [ ${count_f} -eq 4 ] ; then
+          exit1_st=0
+          exit2_st=0
+      else
+          echo "Error---count=${count_f}"
+      fi
+   fi
+   #---
+   if [ ${exit1_st} -eq 0 ] && [ ${exit2_st} -eq 0 ] ; then
+      echo "get mercator finished"
+      # if exist merge files with CDO
+      if [ -s ${DATADIR}/global_analysis_forecast_phy1_${date1}.nc ] ; then
+         echo "cdo -z zip_1 merge global_analysis_forecast_phy{1..4}_${date1}.nc global_analysis_forecast_phy_${date1}.nc"
+         cdo -z zip_1 merge global_analysis_forecast_phy{1..4}_${date1}.nc global_analysis_forecast_phy_${date1}.nc
```

Comment:
You must add ${DATADIR} in front of these filenames, otherwise it will fail if the script is called from another folder than the data is stored.

Reply:
I have added a fix for this